dali/operators/geometry/affine_transforms/transform_base_op.cc

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/operators/geometry/affine_transforms/transform_base_op.h"
+#include "dali/pipeline/data/views.h"
+
+namespace dali {
+
+DALI_SCHEMA(TransformAttr)
+  .DocStr(R"code(Base schema for affine transform generators.)code")
+  .AddOptionalArg("reverse",
```

Comment:
@mzient I wonder if warp should have the option to accept reversed matrix.
Now you have to create two transform ops - one for points and second, with reverse set to True, for images.

Reply:
No. That's not what reverse does. Reverse changes the order of chaining transforms.
A * B instead of B * A.

Reply:
Ok